lib/lotus/action/cookie_jar.rb

```diff
@@ -131,6 +137,7 @@ def extract(env)
         hash   = env[COOKIE_HASH_KEY] ||= {}
         string = env[HTTP_HEADER]
 
+        return if string && string.include?(RACK_SESSION_KEY)
```

Comment:
@AlfonsoUceda this will delete any cookie that contains RACK_SESSION_KEY in either _key_ or _value_

```ruby
# in your action
...
cookies['message'] = "This will be deleted because it contains 'rack.session'"
...
```